system-config/vagrant-reload.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2013 Aidan Nagorcka-Smith
+#
+# MIT License
```

Comment:
I generally do just that -- update the README.md with a little notification about what the code in the repo is licesensed under. I often point to the LICENSE in the repo itself and say "see the LICENSE file for more" or similar. If the code has mixed licenses, I'll say that too, something like "code in this repo is licensed under the MPL 2.0 or other licenses. Please see individual files for more infomation." 

In a Debian package (which is often considered the gold standard of licensing,) they identify each file's copyright and specify each file's license; here's an example: http://metadata.ftp-master.debian.org/changelogs/main/libi/libio-compress-perl/libio-compress-perl_2.066-1_copyright

We don't have to go to that length necessarily, but we do have to acknowledge copyright and license.